app/cdap/components/RecipeList/RecipeTableRow.tsx

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, { useState } from 'react';
+import T from 'i18n-react';
+import moment from 'moment';
+import { IRecipe } from './types';
+import ActionsPopover, { IAction } from 'components/shared/ActionsPopover';
+import MyDataPrepApi from 'api/dataprep';
+import { getCurrentNamespace } from 'services/NamespaceStore';
+import fileDownload from 'js-file-download';
+import ConfirmationModal from 'components/shared/ConfirmationModal';
+import { IState, reset } from './reducer';
+
+interface IRecipeTableRowProps {
+  recipe: IRecipe;
+  createdTimeMillis?: any;
+  isShowAllColumns: boolean;
+  isShowActions: boolean;
+  viewHandler: (selectedRecipe: IRecipe) => void;
+  editHandler: (selectedRecipe: IRecipe) => void;
+  selectHandler: (selectedRecipe: IRecipe) => void;
+  state: IState;
+  dispatch: (action: any) => void;
+}
+
+const PREFIX = 'features.WranglerNewUI.Recipe';
+const DATE_FORMAT = 'MM-DD-YYYY HH:mm A';
+
+export const RecipeTableRow = ({
+  recipe,
+  isShowAllColumns,
+  isShowActions,
+  viewHandler,
+  editHandler,
+  selectHandler,
+  state,
+  dispatch,
+}: IRecipeTableRowProps) => {
+  const [showPopover, setShowPopover] = useState(false);
+  const [isDeleteModalOpen, setDeleteModalOpen] = useState(false);
+  const [isDeleteLoading, setDeleteLoading] = useState(false);
+  const [deleteErrMsg, setDeleteErrMsg] = useState(null);
+  const [extendedDeleteErrMsg, setExtendedDeleteErrMsg] = useState(null);
+
+  const getDateID = (date: Date) => {
+    return moment(date).format(DATE_FORMAT);
```

Comment:
Please use the formatters in [DataFormatter](https://github.com/cdapio/cdap-ui/blob/develop/app/cdap/services/DataFormatter/index.ts)

Reply:
Changed to DataFormatter APi